dlc/src/main/scala/org/bitcoins/dlc/DLCMessage.scala

```diff
@@ -1,46 +1,389 @@
 package org.bitcoins.dlc
```

Comment:
This file has a pretty big diff from the JSON converters, will be good to review